typescript/sdk/src/deploy/router/HyperlaneRouterDeployer.ts

```diff
@@ -105,8 +107,10 @@ export abstract class HyperlaneRouterDeployer<
       }
 
       await super.runIfOwner(local, contracts.router, async () => {
-        const chains = domains.map((id) => DomainIdToChainName[id]);
-        this.logger(`Enroll remote (${chains}) routers on ${local}`);
+        const chains = domains.map((id) => DomainIdToChainName[id] || id);
```

Comment:
Wdyt about a `DomainIdtoChainName` object that gets constructed by the multiprovider and we would query that here?

Reply:
We could, though we'd need to query for any that aren't in the SDK. Doesn't seem worth it since we just use this chains value in the log line right below